src/components/icons/icons.xml

```diff
@@ -498,7 +506,7 @@
       focusable="false"
       viewBox="0 0 512 512">
       <path
-        fill="#188038"
+        fill="currentColor"
```

Comment:
not green ?

Reply:
The icon sometime should be gray (in the topbar, when no filter is in the selection), or even white (in the grid on hover).

I think it's better to set a fill `currentColor` to every icon, and change the font color when we want to change it, rather than setting a default color and overwriting it with a css selector `.o-icon .svg .path{}`.